pkg/install/fixcloudconfig.go

```diff
@@ -0,0 +1,64 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/ghodss/yaml"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
+)
+
+func (i *Installer) fixCloudConfig(ctx context.Context) error {
+	return retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		cm, err := i.kubernetescli.CoreV1().ConfigMaps("openshift-config").Get("cloud-provider-config", metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		var config map[string]interface{}
+		err = yaml.Unmarshal([]byte(cm.Data["config"]), &config)
```

Comment:
json.Unmarshal?

Reply:
as yaml is subset of json, this will work :man_shrugging:  but will change